examples/automm/utils.py

```diff
@@ -0,0 +1,153 @@
+from autogluon.text.automm.constants import (
+    BINARY,
+    MULTICLASS,
+    REGRESSION,
+    ACC,
+    RMSE,
+    CATEGORICAL,
+    NUMERICAL,
+)
+import numpy as np 
+import pandas as pd
+import os
+
+FULLNAME = 'full_name'
+TYPE = 'problem_type'
+METRIC = 'metric'
+
+DATASETS = {
+    'ad': {
+        FULLNAME: 'adult',
+        TYPE: BINARY,
+        METRIC: ACC,
+    },
+    'al': {
+        FULLNAME: 'aloi',
+        TYPE: MULTICLASS,
+        METRIC: ACC,
+    },
+    'ca': {
+        FULLNAME: 'california_housing',
+        TYPE: REGRESSION,
+        METRIC: RMSE,
+    },
+    'co': {
+        FULLNAME: 'covtype',
+        TYPE: MULTICLASS,
+        METRIC: ACC,
+    },
+    'ep': {
+        FULLNAME: 'epsilon',
+        TYPE: BINARY,
+        METRIC: ACC,
+    },
+    'he': {
+        FULLNAME: 'helena',
+        TYPE: MULTICLASS,
+        METRIC: ACC,
+    },
+    'hi': {
+        FULLNAME: 'higgs_small',
+        TYPE: BINARY,
+        METRIC: ACC,
+    },
+    'ja': {
+        FULLNAME: 'jannis',
+        TYPE: MULTICLASS,
+        METRIC: ACC,
+    },
+    'mi': {
+        FULLNAME: 'microsoft',
+        TYPE: REGRESSION,
+        METRIC: RMSE,
+    },
+    'ya': {
+        FULLNAME: 'yahoo',
+        TYPE: REGRESSION,
+        METRIC: RMSE,
+    },
+    'ye': {
+        FULLNAME: 'year',
+        TYPE: REGRESSION,
+        METRIC: RMSE,
+    },
+}
+
+def load_data_per_split(
```

Comment:
Why is this so complicated? Why aren't the datasets in parquet format? How do I access these datasets?

Reply:
Yes. We refine the dataset loading thoroughly. We now support downloading datasets from s3. In README.md, we have one section to describe the accessibility to the data.